Source/Core/Core/HW/WiimoteEmu/Dynamics.cpp

```diff
@@ -120,9 +120,19 @@ void EmulateTilt(RotationalState* state, ControllerEmu::Tilt* const tilt_group,
   const ControlState roll = target.x * MathUtil::PI;
   const ControlState pitch = target.y * MathUtil::PI;
 
+  const auto target_angle = Common::Vec3(pitch, -roll, 0);
+
+  // For each axis, wrap around current angle if target is farther than ~180 degrees.
+  for (std::size_t i = 0; i != target_angle.data.size(); ++i)
+  {
+    auto& angle = state->angle.data[i];
+    if (std::abs(angle - target_angle.data[i]) > MathUtil::PI * 1.001)
```

Comment:
Why are you doing the multiply of 1.001?  Why not just compare against `PI`?

Reply:
When "Angle" was set to 180 tilting would occur in the wrong direction. I assume it's because the floats aren't holding the exact value. I can add a comment.